l10n/fr.json

```
@@ -1,6 +1,11 @@
 { "translations": {
     "Name" : "Nom",
     "URL" : "URL",
+    "Auth" : "Authorisation",
```

```suggestion
    "Auth" : "Autorisation",
```
Or maybe "Authentification" ?